### PR TITLE
Manage specific aws sdk libs

### PR DESCRIPTION
**Pull request by james-michael**
`aws-sdk` is very big bundle of libraries (105 to be exact as of today). It is recommended to use individual sdk gems. Updated to use gems that are used in this client.
Also updated runtime dependencies so that installing this gem pulls dependent gems as well.

### CODE REVIEW

**Comment by james-michael** [comment]
Travis build is failing due to this open issue. https://github.com/rvm/rvm/issues/4210

**Comment by james-michael** [comment]
Travis is having issues downloading "jruby" due to an upstream issue
```
$ rvm use jruby --install --binary --fuzzy
Unknown ruby string (do not know how to handle): jruby-9.1.13.0200.

The command "rvm use jruby --install --binary --fuzzy" failed and exited with 2 during .```

Use a explicit version from http://rubies.travis-ci.org

**Review by tlisonbee — APPROVED** [review]

File: lib/cerberus_utils/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module CerberusUtils
-  VERSION = "2.0.0"
+  VERSION = "2.0.1"
```
Comment (tlisonbee) [inline lib/cerberus_utils/version.rb]: Should it be more than a x.x.1 release?

Comment (sdford) [reply lib/cerberus_utils/version.rb]: I think it should. Even though this fixing a bug I think it should at least be a minor release

**Review by sdford — APPROVED** [review]

File: lib/cerberus_utils/version.rb (hunk 1 above)
Comment (james-michael) [reply lib/cerberus_utils/version.rb]: Ok. updated to be a minor version increment

**Pull request merged**